rust/perspective-viewer/.gitignore

```diff
@@ -0,0 +1 @@
+src/ts/package.json
```

Comment:
Don't add arbitrary files to `.gitignore`, tasks like `twine` and `npm publish` are not aware of these directives which is quite dangerous. If you need to create files as part of the build process, put them in `/build` so they are cleaned and not published - if they need to be published put them in `/dist`.